mettle/configure.ac

```diff
@@ -19,7 +19,7 @@ case $host_os in
 		HOST_OS=apple
 		OBJCFLAGS="$OBJCFLAGS -fobjc-arc"
 		AC_DEFINE(HAVE_WEBCAM)
-		AC_SUBST([PLATFORM_LDADD], ['-framework CoreMedia -framework CoreImage -framework AVFoundation -framework ImageIO'])
+		AC_SUBST([PLATFORM_LDADD], ['-framework AppKit -framework CoreMedia -framework QuartzCore -framework AVFoundation -framework ImageIO'])
```

Comment:
Minor fix: `AppKit` is OSX only so this breaks iOS. https://github.com/bcook-r7/mettle/pull/8 has the fix